elements/lisk-chain/src/state_store/state_store.ts

```diff
@@ -13,20 +13,15 @@
  */
 
 import { codec, Schema } from '@liskhq/lisk-codec';
-import { NotFoundError as DBNotFoundError } from '@liskhq/lisk-db';
+import { IterateOptions, NotFoundError as DBNotFoundError } from '@liskhq/lisk-db';
 import { DB_KEY_STATE_STORE } from '../db_keys';
 import { StateDiff } from '../types';
 import { CacheDB } from './cache_db';
 import { NotFoundError } from './errors';
 import { DatabaseReader, DatabaseWriter } from './types';
 import { copyBuffer } from './utils';
 
-export interface IterateOptions {
-	start: Buffer;
-	end: Buffer;
-	limit?: number;
-	reverse?: boolean;
-}
+export { IterateOptions };
```

Comment:
we need to export it to use it in framework, so that we don't get type `IterateOptions` from DB lib and `StateStore` from chain lib separately as they need to be used together.